<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TevianClient</class>
 <widget class="QMainWindow" name="TevianClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>953</width>
    <height>708</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TevianClient</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Vertical Scroll Bar*/
QScrollBar:vertical {
	border: none;
	background-color: rgb(59, 59, 90);
	width: 20px;
	margin-top: 20px;
	margin-bottom: 20px;
	border-radius: 0px;
}

QScrollBar::handle:vertical{
	background-color: rgb(80, 80, 122);
	min-height: 30px;
	border-radius: 10px;
}

QScrollBar::handle:vertical:hover{
	background-color: rgb(255, 0, 127);
}

QScrollBar::handle:vertical:pressed {
	background-color: rgb(185, 0, 92);
}

QScrollBar::sub-line:vertical {
	border: none;
	background-color: rgb(80, 80, 122);
	height: 20px;
	border-top-left-radius: 10px;
	border-top-right-radius: 10px;
	subcontrol-position: top;
	subcontrol-origin: margin;
}

QScrollBar::sub-line:vertical:hover {
	background-color: rgb(255, 0, 127);
}

QScrollBar::sub-line:vertical:pressed {
	background-color: rgb(185, 0, 92);
}

QScrollBar::add-line:vertical {
	border: none;
	background-color: rgb(80, 80, 122);
	height: 20px;
	border-bottom-left-radius: 10px;
	border-bottom-right-radius: 10px;
	subcontrol-position: bottom;
	subcontrol-origin: margin;
}

QScrollBar::add-line:vertical:hover {
	background-color: rgb(255, 0, 127);
}

QScrollBar::add-line:vertical:pressed {
	background-color: rgb(185, 0, 92);
}

QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
	background: none;
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
	background: none;
}

/* Horizontal Scroll Bar*/
QScrollBar:horizontal {
	border: none;
	background-color: rgb(59, 59, 90);
	height: 20px;
	margin-left: 20px;
	margin-right: 20px;
	border-radius: 0px;
}

QScrollBar::handle:horizontal{
	background-color: rgb(80, 80, 122);
	min-width: 30px;
	border-radius: 10px;
}

QScrollBar::handle:horizontal:hover{
	background-color: rgb(255, 0, 127);
}

QScrollBar::handle:horizontal:pressed {
	background-color: rgb(185, 0, 92);
}

QScrollBar::sub-line:horizontal {
	border: none;
	background-color: rgb(80, 80, 122);
	width: 20px;
	border-top-left-radius: 10px;
	border-bottom-left-radius: 10px;
	subcontrol-position: left;
	subcontrol-origin: margin;
}

QScrollBar::sub-line:horizontal:hover {
	background-color: rgb(255, 0, 127);
}

QScrollBar::sub-line:horizontal:pressed {
	background-color: rgb(185, 0, 92);
}

QScrollBar::add-line:horizontal {
	border: none;
	background-color: rgb(80, 80, 122);
	width: 20px;
	border-top-right-radius: 10px;
	border-bottom-right-radius: 10px;
	subcontrol-position: right;
	subcontrol-origin: margin;
}

QScrollBar::add-line:horizontal:hover {
	background-color: rgb(255, 0, 127);
}

QScrollBar::add-line:horizontal:pressed {
	background-color: rgb(185, 0, 92);
}

QScrollBar::up-arrow:horizontal, QScrollBar::down-arrow:horizontal {
	background: none;
}

QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal {
	background: none;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(59, 59, 90);</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="topMargin">
     <number>9</number>
    </property>
    <property name="rightMargin">
     <number>9</number>
    </property>
    <property name="bottomMargin">
     <number>9</number>
    </property>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="styleSheet">
       <string notr="true">QScrollBar:vertical {
	background-color: rgb(56, 56, 85);
}

QScrollBar:horizontal {
	background-color: rgb(56, 56, 85);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>935</width>
         <height>626</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>9</number>
        </property>
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="topMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <property name="bottomMargin">
         <number>9</number>
        </property>
        <item>
         <widget class="CustomView" name="graphicsView">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustToContents</enum>
          </property>
          <property name="dragMode">
           <enum>QGraphicsView::ScrollHandDrag</enum>
          </property>
          <property name="transformationAnchor">
           <enum>QGraphicsView::NoAnchor</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenDirectory"/>
   <addaction name="actionOpenFiles"/>
   <addaction name="actionFaceReco"/>
   <addaction name="actionPrevFile"/>
   <addaction name="actionNextFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="actionOpenFiles">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/recources/file-folder-data-document-study-paper-explorer-5-21650.png</normaloff>:/resources/recources/file-folder-data-document-study-paper-explorer-5-21650.png</iconset>
   </property>
   <property name="text">
    <string>Open files</string>
   </property>
   <property name="toolTip">
    <string>Open files</string>
   </property>
  </action>
  <action name="actionFaceReco">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/recources/face_reco.png</normaloff>:/resources/recources/face_reco.png</iconset>
   </property>
   <property name="text">
    <string>FaceReco</string>
   </property>
   <property name="toolTip">
    <string>Start face recognition</string>
   </property>
  </action>
  <action name="actionPrevFile">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/recources/arrow_left.png</normaloff>:/resources/recources/arrow_left.png</iconset>
   </property>
   <property name="text">
    <string>PrevFile</string>
   </property>
   <property name="toolTip">
    <string>Previous File</string>
   </property>
  </action>
  <action name="actionNextFile">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/recources/arrow_right.png</normaloff>:/resources/recources/arrow_right.png</iconset>
   </property>
   <property name="text">
    <string>NextFile</string>
   </property>
   <property name="toolTip">
    <string>Next File</string>
   </property>
  </action>
  <action name="actionOpenDirectory">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/recources/folder_open.png</normaloff>:/resources/recources/folder_open.png</iconset>
   </property>
   <property name="text">
    <string>OpenDirectory</string>
   </property>
   <property name="toolTip">
    <string>Open directory with images</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomView</class>
   <extends>QGraphicsView</extends>
   <header>customview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
